frontend/src/scenes/billing/Billing.tsx

```diff
@@ -235,7 +235,7 @@ export function Billing(): JSX.Element {
             </div>
 
             {(products || [])
-                .filter((x) => !x.contact_support)
+                .filter((x) => !x.inclusion_only)
```

Comment:
should it be both `!x.contact_support && !x.inclusion_only`?

Reply:
the only `contact_support` product is also `inclusion_only` so just to keep it simple I did just the one.